xknx/io/udp_client.py

```diff
@@ -64,7 +64,7 @@ def error_received(self, exc):
         def connection_lost(self, exc):
             """Log error. Callback for connection lost."""
             if hasattr(self, 'xknx'):
-                self.xknx.logger.info('closing transport %s', exc)
+                self.xknx.logger.info('Closing transport: %s', exc)
```

Comment:
In my logs `exc` is always None. Maybe we can just remove the parameter? Or did you ever had this set to something else?

Reply:
Always "None" for me as well so far. But I wasn't sure if there might be other scenarios, so I did not touch that.

Reply:
You can delete it IMHO

Reply:
It is `None` when we close the connection or it reaches EOF. 
If the connection gets terminated from the other side there should be an exception.
We can explore this further when handling #349